### DISCUSSION TITLE
Possible to exclude source paths to process?

### DISCUSSION THREAD

**Discussion by brandonkelly** [rectorphp/rector#7030] [Q&A] [answered]
When running `rector process <source>`, is it possible to specify paths that should be _ignored_?

For example, [nystudio107/craft-seomatic](https://github.com/nystudio107/craft-seomatic) has a couple directories that should be skipped by Rector:

- [src/integrations](https://github.com/nystudio107/craft-seomatic/tree/develop/src/integrations) (contains classes that extend parent classes provided by third party plugins, not present in the source repo or its dependencies)
- [src/models/jsonld](https://github.com/nystudio107/craft-seomatic/tree/develop/src/models/jsonld) (contains generated classes that shouldn’t be modified directly)

**Comment by TomasVotruba** [comment, marked as answer]
You can define this in `rector.php` - https://github.com/rectorphp/rector/blob/main/docs/how_to_ignore_rule_or_paths.md

**Reply by brandonkelly** [reply]
Thanks!

**Reply by petersendidit** [reply]
Updated link that works https://getrector.com/documentation/ignoring-rules-or-paths
